Misc/ACKS

```diff
@@ -1896,3 +1896,4 @@ Robert Leenders
 Tim Hopper
 Dan Lidral-Porter
 Ngalim Siregar
+Vlad Emelianov
```

Comment:
Please resolve merge conflict here

Reply:
Will do

Reply:
Resolved.

Reply:
Doesn't look resolved, can you please check this again.